converters/tokenMetaData.go

```diff
@@ -59,3 +62,32 @@ func whiteListedStorage(uris [][]byte) bool {
 
 	return strings.HasPrefix(string(uris[0]), ipfsURL)
 }
+
+// PrepareNFTUpdateData will prepare nfts update data
+func PrepareNFTUpdateData(buffSlice *data.BufferSlice, updateNFTData []*data.UpdateNFTData, accountESDT bool) error {
+	for _, nftUpdate := range updateNFTData {
+		id := nftUpdate.Identifier
+		if accountESDT {
+			id = fmt.Sprintf("%s-%s", nftUpdate.Address, nftUpdate.Identifier)
+		}
+
+		metaData := []byte(fmt.Sprintf(`{"update":{"_id":"%s", "_type": "_doc"}}%s`, id, "\n"))
+		base64Attr := base64.StdEncoding.EncodeToString(nftUpdate.NewAttributes)
+		serializedData := []byte(fmt.Sprintf(`{"script": {"source": "if (ctx._source.containsKey('data')) {ctx._source.data.attributes = params.attributes}","lang": "painless","params": {"attributes": "%s"}}, "upsert": {}}`, base64Attr))
+		if len(nftUpdate.URIsToAdd) != 0 {
+			marshalizedURIS, err := json.Marshal(nftUpdate.URIsToAdd)
+			if err != nil {
+				return err
+			}
+
+			serializedData = []byte(fmt.Sprintf(`{"script": {"source": "if (ctx._source.containsKey('data')) { if (!ctx._source.data.containsKey('uris')) { ctx._source.data.uris = params.uris; } else {  ctx._source.data.uris.addAll(params.uris); }}","lang": "painless","params": {"uris": %s}},"upsert": {}}`, marshalizedURIS))
```

Comment:
so there is any URI, then `serializedData` will be overwritten. this means:
if hasAttributes && hasURIs { setURIs }
if hasAttributes && !hasURIs { setAttributes }

is this ok ?

Reply:
if we have URIs ---> will be added to the current list
if we have attributes ---> will be overwritten